extensions/wikia/VisualEditorTourExperiment/scripts/VisualEditorTourExperimentInit.js

```diff
@@ -3,25 +3,31 @@ define('VisualEditorTourExperimentInit',
 	function ($, VETour, veTourConfig, abTest, tracker) {
 		'use strict';
 
-		var experimentName = 'contribution-experiments',
-			freshlyRegisteredExperimentId = 5654433460,
-			usersWithoutEditExperimentId = 5735670451;
+		var experimentName = 'contribution-experiments';
 
 		function init() {
+			var lang = mw.config.get('wgUserLanguage');
 			if (isEnabled()) {
 				clearEntrypointPopover();
-				(new VETour(veTourConfig)).start();
+				(new VETour(veTourConfig[lang])).start();
 			}
 		}
 
 		function isEnabled() {
-			return isExperimentVariation() &&
-				(isNewlyregistered() || isUserwithoutedit()) &&
-				!$.cookie('vetourdisabled');
+			var enable = isAllowedCommunity();
+			if (mw.config.get('wgUserName') != null) {
+				enable = enable && isUserLanguagePreferenceJapanese() && isUserwithoutedit();
+			}
+			return enable && !$.cookie('vetourdisabled');
 		}
 
 		function trackPublish() {
-			if (isExperimentVariation() && (isNewlyregistered() || isUserwithoutedit())) {
+			var enable = isAllowedCommunity();
+			if (mw.config.get('wgUserName') != null) {
+				enable = enable && isUserLanguagePreferenceJapanese() && isUserwithoutedit();
+			}
+			if (enable) {
+				debugger;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this debugger statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ADebuggerStatement)